lib/compile-ember-features.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+var Funnel = require('broccoli-funnel');
+var CompileFeaturesJSON = require('./utils/compile-features-json');
+
+module.exports = function compileEmberFeatures(environment) {
+  var featuresJSON = new Funnel('.', {
```

Comment:
We shouldn't attempt to watch `.` like this (it can cause recursive rebuilds due to `tmp` changing).